build_tools/cmake/rebuild.sh

```diff
@@ -35,6 +35,12 @@ else
   mkdir build
 fi
 cd build
-"$CMAKE_BIN" -G Ninja -DCMAKE_BUILD_TYPE=FastBuild -DIREE_BUILD_COMPILER=ON -DIREE_BUILD_TESTS=ON -DIREE_BUILD_SAMPLES=OFF -DIREE_BUILD_DEBUGGER=OFF -DIREE_BUILD_PYTHON_BINDINGS=ON ..
+"$CMAKE_BIN" -G Ninja -DCMAKE_BUILD_TYPE=FastBuild \
+                      -DIREE_BUILD_COMPILER=ON \
+                      -DIREE_BUILD_TESTS=ON \
+                      -DIREE_BUILD_SAMPLES=OFF \
+                      -DIREE_BUILD_DOCS=OFF \
```

Comment:
I actually wonder if we should turn this on in the CI... How much time does it add? We really need to split the CI into multiple builds so we don't have to make this either-or decision.

Reply:
Turned on for now. Generally I would prefer to have small and dedicated tests instead of one-test-for-all. But I guess configuration is kinda different. So I'm fine here.

Reply:
Agreed, that's what I meant about splitting the CI :-)